defaults/main.yml

```diff
@@ -23,6 +23,10 @@ __slurm_user_name: "{{ (slurm_user | default({})).name | default('slurm') }}"
 # TODO: this could be incorrect, use the group collection from galaxyproject.galaxy
 __slurm_group_name: "{{ (slurm_user | default({})).group | default(omit) }}"
 
+
+# install from custom debian repos
+slurm_configure_repos: false
+
```

Comment:
The file seems to declare fixed variables in the first part, then dynamic variables (starting with `__`).
I would move this new variable to the first part, above `__slurm_user_name`.

Reply:
fixed